### PR TITLE
[Feature] Support show binlog properties by show create table

### PR DESCRIPTION
**Pull request by smartlxh**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

supporting show binlog propeties by sql `show create table table_name`

image

image

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Review by murphyatwork — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 2 / 19 (10.53%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|18|05.56%|[2289, 2290, 2291, 2292, 2293, 2295, 2296, 2297, 2298, 2300, 2301, 2302, 2303, 2305, 2306, 2307, 2308]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|1|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 1]
```diff
@@ -2157,11 +2157,12 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
                         false);
         olapTable.setEnablePersistentIndex(enablePersistentIndex);
 
-        boolean enableBinlog = PropertyAnalyzer.analyzeBooleanProp(properties, PropertyAnalyzer.PROPERTIES_BINLOG_ENABLE, false);
```
Comment (caneGuy) [inline fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:2160]: add some description in the pr summary

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 2]
```diff
@@ -2157,11 +2157,12 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
...
         if (properties != null && (properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_ENABLE) ||
                 properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_MAX_SIZE) ||
                         properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_TTL))) {
             try {
+                boolean enableBinlog = PropertyAnalyzer.analyzeBooleanProp(properties,
+                        PropertyAnalyzer.PROPERTIES_BINLOG_ENABLE, false);
```
Comment (caneGuy) [inline fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:2165]: refine log below

File: fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java [hunk 3]
```diff
@@ -2283,6 +2283,31 @@ public static void getDdlStmt(String dbName, Table table, List<String> createTab
                     sb.append(olapTable.enableReplicatedStorage()).append("\"");
                 }
 
+                // binlog config
+                if (olapTable.isHaveBinlogConfig()) {
```
Comment (caneGuy) [inline fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java:2287]: isHaveBinlogConfig -> containsBinlogConfig

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 4]
```diff
@@ -2157,11 +2157,12 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
                         false);
         olapTable.setEnablePersistentIndex(enablePersistentIndex);
 
-        boolean enableBinlog = PropertyAnalyzer.analyzeBooleanProp(properties, PropertyAnalyzer.PROPERTIES_BINLOG_ENABLE, false);
         if (properties != null && (properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_ENABLE) ||
                 properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_MAX_SIZE) ||
                         properties.containsKey(PropertyAnalyzer.PROPERTIES_BINLOG_TTL))) {
```
Comment (caneGuy) [inline fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:2162]: nit: i think we should check binlog_enable first

**Review by caneGuy — CHANGES_REQUESTED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 9 / 27 (33.33%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|18|05.56%|[2289, 2290, 2291, 2292, 2293, 2295, 2296, 2297, 2298, 2300, 2301, 2302, 2303, 2305, 2306, 2307, 2308]|
|:large_blue_circle:|com/starrocks/binlog/BinlogManager.java|2|3|66.67%|[78]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/leader/ReportHandler.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/alter/SchemaChangeHandler.java|1|1|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java (hunk 2 above)
Comment (smartlxh) [reply fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java]: I will fix this in another pr

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 2 / 19 (10.53%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|1|18|05.56%|[2289, 2290, 2291, 2292, 2293, 2295, 2296, 2297, 2298, 2300, 2301, 2302, 2303, 2305, 2306, 2307, 2308]|
|:large_blue_circle:|com/starrocks/alter/SchemaChangeHandler.java|1|1|100.00%|[]|

**Review by caneGuy — APPROVED** [review]

**Review by murphyatwork — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by smartlxh** [comment]
run starrocks_admit_test

**Comment by smartlxh** [comment]
run starrocks_admit_test

**Pull request merged**